packages/pods-core/src/v2/PodConfigManager.ts

```diff
@@ -62,9 +61,10 @@ export class PodV2ConfigManager {
         fPath,
       });
 
-      if (isExportPodConfigurationV2(config)) {
-        configs.push(config);
-      }
+      // if (isExportPodConfigurationV2(config)) {
+      //   configs.push(config);
+      // }
+      configs.push(config);
```

Comment:
Don't leave the commented code in - let's just delete it. 

Can we also add a TODO - we should give some UI feedback to a user if their configuration has issues that makes it not conform to what a V2 config requires.  Either
- perhaps we can even have some visual cue within the quick pick itself (like a detail that says "Config missing properties: podId"
- When the config is open, have some sort of code warning.